CliFx.Tests/Services/HelpTextRendererTests.cs

```diff
@@ -106,6 +106,23 @@ private static IEnumerable<TestCaseData> GetTestCases_RenderHelpText()
                     "-h|--help", "Shows help text."
                 }
             );
+
+            yield return new TestCaseData(
+                CreateHelpTextSource(
+                    new[] { typeof(SampleValueOptionCommand) },
+                    typeof(SampleValueOptionCommand)),
+
+                new[]
+                {
+                    "Description",
+                    "SampleValueOptionCommand description.",
+                    "Usage",
+                    "testapp sampleval", "-f option_f_value", "[options]",
```

Comment:
Also, should the `[options]` be shown if all options (except `--help` and `--version`) are required? I think it's better to not show it in this case, but it doesn't hurt anyone if it's there.

Reply:
Thanks for pointing this out. initially I didn't realized that the `[options]` tag doesn't makes sense when all non default options are required.

I added this behavior to the PR and related test cases.